common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/Indexer.scala

```diff
@@ -31,8 +30,13 @@ object Indexable {
     new Indexable[AugmentedImage] {
       def id(image: AugmentedImage) =
         image.id.canonicalId
+
       def version(image: AugmentedImage) =
-        image.version
+        10 * (image.version + image.source.version) + (image.source match {
```

Comment:
Why do we multiply by 10?

Reply:
Bleurgh, knew I'd forgotten something - same reason as the very detailed comment on the works indexer, I will clarify this